index.html

```diff
@@ -2083,34 +2084,59 @@ <h3 id="applying">
           </li>
           <li>If <var>src</var> is <code>undefined</code>, or if the result of
           running <a>Is origin potentially trustworthy</a>
-          with the origin of <var>src</var> is <code>Not Trusted</code>, return
-          <code>undefined</code>.
+          with the origin of <var>src</var> is <code>Not Trusted</code>, abort
+          these steps and return <code>undefined</code>.
           </li>
           <li>Otherwise, let <var>registration</var> be an object with
-          properties <code>src</code> and <code>scope</code>. All properties
-          initially set to <code>undefined</code>.
+          properties <code>src</code>, <code>scope</code>, <code>type</code> and
+          <code>use_cache</code>. All properties initially set to
+          <code>undefined</code>.
           </li>
           <li>Set <var>registration</var>'s <code>src</code> property to be
           <var>src</var>.
           </li>
-          <li>Let <var>type</var> be the result of running the <a>steps
+          <li>Let <var>scope</var> be the result of running the <a>steps
```

Comment:
I understood the intention. Please check if it actually doesn't attempt to register in that case (i.e. when the sub algorithm returns _undefined_.) I left a comment here: https://github.com/w3c/manifest/pull/507#discussion_r92408719.